<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcLauncher</class>
 <widget class="QMainWindow" name="ProcLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cardinal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Exec</string>
          </property>
          <property name="buddy">
           <cstring>inpProcName</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="inpProcName">
          <property name="toolTip">
           <string>Excecutable name (e.g. /usr/bin/ls)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Args</string>
          </property>
          <property name="buddy">
           <cstring>inpProcArgs</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="inpProcArgs">
          <property name="toolTip">
           <string>Arguments to the program</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bFindExec">
        <property name="text">
         <string>Find...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="cbMeasurementLabel">
        <property name="toolTip">
         <string>Label under which measurement will be filed</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Measurement label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbDefer">
        <property name="toolTip">
         <string>Defer tracing until &quot;start&quot; pressed</string>
        </property>
        <property name="text">
         <string>Defer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Connection</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lConnectionStatus">
           <property name="text">
            <string>Connection status</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="bReconnect">
           <property name="text">
            <string>Reconnect</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboSelectedDevice"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Process launch</string>
        </property>
        <layout class="QGridLayout" name="procLayout"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="deviceOpLayout">
      <property name="title">
       <string>Device operations</string>
      </property>
      <layout class="QGridLayout" name="deviceLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSetup_device"/>
    <addaction name="actionCollect_cores"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSetup_device">
   <property name="text">
    <string>Setup device</string>
   </property>
  </action>
  <action name="actionCollect_cores">
   <property name="text">
    <string>Collect cores</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
